systems/plants/URDFRigidBodyManipulator.cpp

```diff
@@ -337,6 +339,7 @@ bool URDFRigidBodyManipulator::addURDF(boost::shared_ptr<urdf::ModelInterface> _
       // pitch is irrelevant
       bodies[index]->Ttree = Matrix4d::Identity();
       bodies[index]->T_body_to_joint = Matrix4d::Identity();
+      bodies[index]->setJoint(std::unique_ptr<RollPitchYawFloatingJoint>(new RollPitchYawFloatingJoint(jointname, Isometry3d::Identity())));
```

Comment:
Previously I wasn't making a DrakeJoint for bodies with an rpy parameterized floating joint.